Add Spanish README

UnixY2K opened the pull request:
this PR adds a new readme based on the main readme

NancyLi1013 committed:
[fluidsynth/redis-plus-plus] Update to the latest released version (#12974)

strega-nil committed:
remove registries.h and fwd/registries.h... (#13017)

ftheile committed:
Fix bullet lists for integration.md on readthedocs.io (#12873)

JackBoosY committed:
[openmpi/vcpkg_build_make] Disable parallel build (#12975)

jherico committed:
[openxr-loader] Update to latest version, official repo (#12060)

Co-authored-by: NancyLi1013 <46708020+NancyLi1013@users.noreply.github.com>

myd7349 committed:
[numcpp] Add new port (#12682)

myd7349 committed:
[cmocka] Update to 2020-08-01 (#12828)

evpobr committed:
[libvorbis] Update to 1.3.7 (#12909)

g-maxime committed:
[libmediainfo] Update to 20.08 (#12944)

NancyLi1013 committed:
[sqlite-orm] Update to latest version (#12999)

albertziegenhagel committed:
[openblas] Fix generating pkg-config file (#12956)

PhoebeHui committed:
[vcpkg] fix hint message (#13043)

LilyWangL committed:
[x265] Update source and version (#13048)

Co-authored-by: NancyLi1013 <46708020+NancyLi1013@users.noreply.github.com>

eao197 committed:
RESTinio updated to v.0.6.10. (#13018)

an-tao committed:
[drogon] Fix building errors on Windows and Linux (#13007)

Co-authored-by: Jack·Boos·Yu <47264268+JackBoosY@users.noreply.github.com>

stekyne committed:
[portmidi] Updated portmidi package to use alternative code source (#12046)

Co-authored-by: Billy Robert O'Neal III <bion@microsoft.com>
Co-authored-by: nicole mazzuca <mazzucan@outlook.com>

RockinRoel committed:
[wt] Update to 4.4.0 (#13005)

stoperro committed:
[dlib] Update to v19.21 (#13061)

JonLiu1993 committed:
[libjpeg-turbo] Update to 2.0.5 (#12973)

UnixY2K committed:
[typo] Fix incorrect Path in readme  (#13066)

seanyen committed:
[behaviortree-cpp] Add new port (#12769)

Co-authored-by: nicole mazzuca <mazzucan@outlook.com>

fquinner committed:
[openmama] Feature use cmake for openmama (#13034)

PhoebeHui committed:
[gdal] Fix static build issue (#12541)

jozefizso committed:
[protobuf] protobuf v3.13.0 (#12918)

cenit committed:
[ffmpeg] fix cmake module (#13004)

FranciscoPombal committed:
[libtorrent] bump version and adjust python patch (#12946)

Ghabry committed:
[fluidlite] Add new port (#13006)

JackBoosY committed:
[pcl] check SSE flags (#13020)

* [pcl] Always check SSE flags

* check sse if sse enabled

NancyLi1013 committed:
[boringssl] Fix build error on unix (#13056)

LilyWangL committed:
[berkeleydb] Fix build error with Visual Studio 16.7.0 (#12870)

JackBoosY committed:
[boost-uninstall] Add port to easily uninstall all boost components (#12851)

Co-authored-by: Daniel Marshall <xandan@gmail.com>

RT2Code committed:
[imgui] Update to 1.78 (#13064)

https://github.com/ocornut/imgui/releases/tag/v1.78

longnguyen2004 committed:
[many ports] Fix various problems with mingw (#12886)

vicroms reviewed (changes requested):
I'm still half-way through reviewing this. It would be useful to run the text through Word or (Google Docs) to correct missing diacritics and exclamation/interrogation marks.

jozefizso committed:
[grpc] grpc v1.30.2 (#12920)

ras0219 committed:
[vcpkg] Fix editable bug (#12910)

ras0219 committed:
[sfml][imgui-sfml] Push SFML_STATIC_LIBRARIES setting into `sfml` (#11800)

strega-nil committed:
[vcpkg.cmake] check for vcpkg.json only in `${CMAKE_SOURCE_DIR}` (#12523)

hanst99 committed:
[vcpkg] Do not use raw paths inside regexes (#12964)

Co-authored-by: Billy Robert O'Neal <bion@microsoft.com>

Hoikas reviewed (commented):
My Spanish is entirely too rusty to review any particulars around grammar, but this is what I noticed.

ras0219 committed:
[vcpkg] Hotfix regression in #12523 (#13092)

Co-authored-by: Robert Schumacher <roschuma@microsoft.com>

BillyONeal committed:
Sort file lists and failure logs together. (#13067)

jozefizso committed:
[grpc][upb][re2] grpc v1.31.1 (#13084)

strega-nil committed:
[behaviortree-cpp] add REQUIRED to find_package calls (#13087)

Co-authored-by: ras0219 <533828+ras0219@users.noreply.github.com>

janisozaur committed:
[vcpkg] Use https sources for common tools (#12738)

PhoebeHui committed:
[sqlitecpp] Fix dependency port (#13108)

JonLiu1993 committed:
[Box2d] Update to 2.4.0 (#13058)

RT2Code committed:
[josuttis-jthread] Fix include path (#13040)

jgehw committed:
fix vcpkg_configure_cmake for case when having semicolons in OPTIONS (#12977)

strega-nil committed:
[libigl] fix imgui feature (#12691)

mfornace committed:
[cpp-taskflow] Update to 2.5.0 (#12923)

Co-authored-by: JackBoosY <yuzaiyang@beyondsoft.com>

huahang committed:
[vcpkg] Fix build on old compilers. (#12950)

SupSuper committed:
Fix MSYS2 keyring not verifying on path with spaces (#13104)

NancyLi1013 committed:
[opencolorio] Update repository (#12869)

luncliff committed:
[openssl-unix] use xcrun for iOS triplet build (#12527)

* [openssl] use xcrun for iOS triplet build

* config for iOS simulator architecture
* use .dylib suffix for iOS shared is used

* [openssl-unix] increase port version in CONTROL

* Update CONTROL

* [openssl-unix] update port version

* prevent collision with #12544

* [openssl-unix] make disable platform specific

* [openssl-unix] replace set to list(APPEND)

Co-authored-by: Lily <47812810+LilyWangL@users.noreply.github.com>

BillyONeal committed:
Disable fluidlite in CI. (#13125)

* Disable fluidlite in CI.

Fluidlite, added 3 days ago in https://github.com/microsoft/vcpkg/pull/13006 , tries to install the same files as fluidsynth; it's similar to the 'boringssl' situation. Given that fluidlite doens't have any dependencies, this change disables that one rather than fluidsynth,

* Add warning message.

karjonas committed:
[irrlicht] Patch moved sysctl.h header file (#13137)

Sysctl has been deprecated for a while and in newer versions of glibc it
is removed which leads to a compilation error on modern Linux. This is
fixed in SVN trunk of irrlich so I copied the include guards from there.
In this case sysctl headers are only loaded for OS X where they are also
actually used.

BillyONeal committed:
[vcpkg] Fix #13094 by avoiding the comma operator. (#13117)

* Fix #13094 by avoiding the comma operator.

Drive-by remove unused parenthesis and remove macro inversion in metrics.cpp.

JackBoosY committed:
[VMSS] Fix the printed error message (#13107)

* check error message

* Remove https://

* Revert test changes

strega-nil committed:
[vcpkg] Fix upgrade for port-version (#13069)

Co-authored-by: Billy Robert O'Neal <bion@microsoft.com>

RT2Code committed:
[directxtk][directxtk12][directxtex][directxmesh] Update to latest version (#13060)

ras0219 committed:
[vcpkg_find_acquire_program] Cleanup and add CI testing (#12982)

evpobr committed:
[libsndfile] Update to 1.0.29 (#12921)

Co-authored-by: Billy Robert O'Neal III <bion@microsoft.com>

JackBoosY committed:
[libusb] Add vcpkg-cmake-wrapper (#13057)

Neumann-A committed:
[lapack-reference] fix cblas feature (#12929)

BillyONeal committed:
Allow use of VS LLVM clang-format. (#13142)

JackBoosY committed:
[clapack] Ignore check of lapack:x64-osx in the baseline (#13148)

BillyONeal committed:
[skia, ompl, capnproto] Update VMs for 2020-08-12 (including VS 16.7) (#12884)

Co-authored-by: Nicole Mazzuca <mazzucan@outlook.com>

jgehw committed:
[vcpkg] Disable timeout (default 10s) in vcpkg_acquire_msys (#13086)

JackBoosY committed:
[rpclib] Disable parallel configure (#13162)

JackBoosY committed:
[bigint/zxing-cpp] Adapter xcode 5.1.1 and set it as a dependency of zxing-cpp (#13129)

* [bigint] Adapter xcode 5.1.1

* [bigint] Add vcpkg-cmake-wrapper

* [zxing-cpp] use vcpkg bigint instead

Alex-PLACET committed:
[exiv2] Add video support feature (#13167)

JackBoosY committed:
[vcpkg_from_sourceforge] Skip mirrors that are in 'disaster recovery' mode (#13176)

cenit committed:
[OpenMVG/OpenMVS] fix tools (#12229)

NancyLi1013 committed:
[cspice] Add 64bit packages (#12903)

ekilmer committed:
[rapidcheck] Update and fix compiler error with clang-10 (#13143)

Neumann-A committed:
[expat] Fix a few details with expat.  (#13022)

Co-authored-by: NancyLi1013 <46708020+NancyLi1013@users.noreply.github.com>

albertziegenhagel committed:
[vtk] Fix building with MPI and Python enabled (#12971)

GPSnoopy committed:
[Arrow] Update to 1.0.1 (#13165)

marekr committed:
[tcl] Install tcl shell to tools (#12987)

Co-authored-by: Lily <47812810+LilyWangL@users.noreply.github.com>

BillyONeal committed:
[skia] Remove use of vcpkg_apply_patches. (#13186)

Co-authored-by: wangli28 <wangli28@beyondsoft.com>

Kwizatz committed:
[v8] Add v8 Javascript engine port (#372). (#12687)

NancyLi1013 committed:
[embree3] Fix usage in static build (#13182)

Ghabry committed:
[FluidLite] Bump to latest version (#13181)

daniele77 committed:
[cli] Update the version to 1.2.1 (#13180)

Sibras committed:
[bzip2] Update to 1.0.8. (#13178)

Fixes #12960

KevinZonda committed:
[Readme] fix typo & sync content (#13237)

cenit committed:
[OpenMVS] add missing include directories for downstream projects (#13233)

ticking0 committed:
[rest-rpc]add new port (#13232)

n4z4m3 committed:
[gdal] for static builds utilize static proj library (#13226)

Commit:
[bfgroup-lyra] Upgrade to version 1.5 (latest) (#13222)

stevemk14ebr committed:
[polyhook2] Update to latest (#13221)

Sibras committed:
[libvpx] Update to 1.9.0. (#13213)

SpriteOvO committed:
Fix link (#13209)

Commit:
[toml11] Update to version 3.5.0 (latest) (#13204)

leha-bot committed:
[matplotlib-cpp] Upgrade to latest version (#13201)

myd7349 committed:
[cpu-features] Add new port (#12681)

rkesters committed:
[caf] Update caf to 0.17.6 (#13175)

Co-authored-by: Phoebe <20694052+PhoebeHui@users.noreply.github.com>
Co-authored-by: Billy Robert O'Neal III <bion@microsoft.com>

JonLiu1993 committed:
[libarchive] Update to 3.4.3 (#13179)

NancyLi1013 committed:
[clapack, lapack-reference] Add conflict messages (#13200)

Co-authored-by: Jack·Boos·Yu <47264268+JackBoosY@users.noreply.github.com>
Co-authored-by: Billy Robert O'Neal III <bion@microsoft.com>

rkitover committed:
[ffmpeg] Use nproc - 1 build jobs for ffmpeg instead of 6. (#12659)

oswjk committed:
[geos] Update to 3.8.1 (#12163)

Hoikas committed:
[libhsplasma] Add new port (#13174)

Co-authored-by: Jack·Boos·Yu <47264268+JackBoosY@users.noreply.github.com>
Co-authored-by: Billy Robert O'Neal III <bion@microsoft.com>

AenBleidd committed:
[sentry-native] Add new port (#13118)

Signed-off-by: Vitalii Koshura <lestat.de.lionkur@gmail.com>

rioki committed:
[libxmlmm ] Add new port libxmlmm 0.6.0 (#13139)

ras0219 committed:
[vcpkg_configure_cmake] Pass CMAKE_DISABLE_SOURCE_CHANGES to all ports by default (#12846)

* [vcpkg] Pass CMAKE_DISABLE_SOURCE_CHANGES to all ports by default

* [docs] Regenerate

Co-authored-by: Robert Schumacher <roschuma@microsoft.com>
Co-authored-by: Billy Robert O'Neal III <bion@microsoft.com>

ras0219 committed:
[vcpkg] Replace uses of msys pacman.exe with direct package downloads (#13019)

* wip

Apply suggestions from code review

Co-authored-by: Alexander Neumann <30894796+Neumann-A@users.noreply.github.com>

* [vcpkg_acquire_msys] Trim list of packages. Introduce defaults.

* [behaviortree-cpp] Fix dynamic dependency on ZMQ

Co-authored-by: Robert Schumacher <roschuma@microsoft.com>
Co-authored-by: Alexander Neumann <30894796+Neumann-A@users.noreply.github.com>
Co-authored-by: Billy Robert O'Neal III <bion@microsoft.com>

ras0219 committed:
[vcpkg] Implement 'repository' tag for NuGet binary caching (#13228)

This tag is required to correctly interface with GitHub Packages, because GHP shares a single remote for the entire organization and uses the repository field to associate a package with a particular repository.

This tag will be automatically generated if the default GitHub Actions environment variables are present (GITHUB_XYZ) or if the user defines VCPKG_NUGET_REPOSITORY.

Co-authored-by: Robert Schumacher <roschuma@microsoft.com>

ajaspe committed:
[levmar] Add new port (#13164)

Co-authored-by: LilyWangL <wangli28@beyondsoft.com>
Co-authored-by: Billy Robert O'Neal III <bion@microsoft.com>

ZeeWanderer committed:
[blend2d] Update to beta_2020-08-24 (#13258)

StarGate-One committed:
[bzip2] Add SHA512 hash check (#13242)

cenit committed:
[darknet] update to v2020-08-30 (#13238)

grdowns committed:
[vcpkg] Add July 2020 changelog (#13302)

* Create July + half of June changelog

* Update test numbers; rephrase items

* Correct release notes begin date

* Remove false entries

BillyONeal committed:
[json5-parser] update source for json5-parser from BitBucket to GitHub. (#13289)

Co-authored-by: NancyLi1013 <46708020+NancyLi1013@users.noreply.github.com>

JackBoosY committed:
[vcpkg baseline][rest-rpc] Skip check in baseline (#13295)

strega-nil committed:
[vcpkg] Registries MVP (#13038)

strega-nil committed:
[vcpkg RFC] initial registries RFC (#12881)

albertziegenhagel committed:
[hypre] Update to 2.19.0 (#13252)

Co-authored-by: wangli28 <wangli28@beyondsoft.com>

StarGate-One committed:
Correct Spelling of Port-Verison to Port-Version in CONTROL file (#13315)

strega-nil committed:
[vcpkg.cmake] error out if the mode has changed (#13313)

PhoebeHui committed:
[many ports] Add supports (#12857)

Co-authored-by: Billy Robert O'Neal <bion@microsoft.com>

psfrolov committed:
11129: Add VCPKG_LINKER_FLAGS_<CONFIG>. (#11290)

Co-authored-by: Nicole Mazzuca <mazzucan@outlook.com>
Co-authored-by: JackBoosY <yuzaiyang@beyondsoft.com>
Co-authored-by: Billy Robert O'Neal III <bion@microsoft.com>

myd7349 committed:
[simdjson] Update to 0.5.0 (#13079)

kibaamor committed:
[knet] Add new port (#13207)

petersteneteg committed:
[freetype] static build fix (#13250)

JonLiu1993 committed:
[folly] Fix findpackage error messages (#13278)

Co-authored-by: NancyLi1013 <46708020+NancyLi1013@users.noreply.github.com>

hkaiser committed:
Adapting for HPX V1.5 (#13309)

BullyWiiPlaza committed:
[cli11] Update to 1.9.1 (#13280)

ras0219 committed:
[vcpkg] Delay failures in `vcpkg install` until missing manifest requested (#13231)

rkitover committed:
Fix ffmpeg build for PATH with spaces. (#13275)

Followup on 085225b7b.

Signed-off-by: Rafael Kitover <rkitover@gmail.com>

PhoebeHui committed:
[dlib] Fix dependency port sqlite3 (#13325)

coryan committed:
[google-cloud-cpp] Upgrade to v1.17.1 (#13336)

FranciscoPombal committed:
[libtorrent] Update to 1.2.10 (#13341)

StarGate-One committed:
[vcpkg_acquire_msys.cmake] Add gettext tool (#13297)

 1. Added dependencies to gettext-0.19.8.1-1-x86_64.pkg.tar.xz
    of libintl libgettextpo libasprintf
 2. Added dependency download for libgettextpo-0.19.8.1-1-x86_64.pkg.tar.xz
 3. Added dependency download for libasprintf-0.19.8.1-1-x86_64.pkg.tar.xz
 4. Put downloads in reverse dependcy order - seems it is called
    topological order now

strega-nil committed:
fix compile on g++ 6 (#13290)

Chronial committed:
[sqlitecpp] Update to 3.1.1 (#13334)

ras0219 committed:
[blaze][ceres][clapack][geogram][lapack][lapack-reference][opencv4][selene] Fix deps issues (#13229)

Co-authored-by: Robert Schumacher <roschuma@microsoft.com>
Co-authored-by: Billy Robert O'Neal III <bion@microsoft.com>

JackBoosY committed:
[sentry-native] Fix build failure and cmake path (#13329)

* [sentry-native] Fix build failure and cmake path

* Re-fix the static_assert issue

mcmtroffaes committed:
[fmt] Fix build error when build type is release. (#13352)

cjdb committed:
[constexpr-contracts] s/HEAD_REF master/HEAD_REF main/ (#13357)

AenBleidd committed:
[rappture] Add new port (#13321)

NancyLi1013 committed:
[sqlite-orm] Add features and also fix usage (#13328)

Sibras committed:
[libxml2] Update to 2.9.10 (#13266)

AenBleidd committed:
[sentry-native] Add ability to control SENTRY_BACKEND variable (#13362)

kevinlul committed:
[curl] Update to 7.22.0 (#13045)

DomClark committed:
[serd] Add new port (#13260)

geiseri committed:
Add option for WCHAR32 support in ImGUI (#13285)

JackBoosY committed:
[python3] Fix installation on OpenSUSE (#13274)

ManuelKugelmann committed:
Fix [boost] MSVC 2013 v120 Build (#10645)

JackBoosY committed:
[powershell] Update to 7.0.3 (#13267)

ZeeWanderer committed:
[highfive] Update to 2.2.2 (#13257)

ThomsonTan committed:
Download pkgconfig for protobuf in download mode (#13263)

* Download pkgconfig for protobuf in download mode

* Bump version of protobuf

* Update ports/protobuf/CONTROL

Co-authored-by: Jack·Boos·Yu <47264268+JackBoosY@users.noreply.github.com>

dan-shaw committed:
[vcpkg] json format for search (#12509)

xieyubo committed:
Add community triple arm64-linux and arm-linux. (#11880)

Verify the following command on both 32 bits and 64 bits OS on Raspberry pi 3.

    ./bootstrap-vcpkg.sh
    ./vcpkg install boost

vejmartin committed:
[chromium-base] Add new port (#12396)

Co-authored-by: JackBoosY <yuzaiyang@beyondsoft.com>
Co-authored-by: Huahua <ihuahua@live.com>

lazyhamster committed:
[vcpkg] Fixed 'update' command output (#13378)

here-mikelley committed:
[pcl] Move VTK to an optional feature (#10449)

Co-authored-by: Jack·Boos·Yu <47264268+JackBoosY@users.noreply.github.com>

Vennor committed:
[entt] Update to 3.5.1 (#13318)

AenBleidd committed:
[sentry-native] Update to 0.4.1 (#13305)

Signed-off-by: Vitalii Koshura <lestat.de.lionkur@gmail.com>

LilyWangL committed:
[realsense2] Update to 2.38.1 (#13273)

BillyONeal committed:
[xtensor-io] [taskflow] [tcl] [sophus] Fix CI 2020-09-06 (#13382)

* [xtensor-io] Disable undeclared HighFive detection which is currently broken.

HIGHFIVE 2.2.2: (Re)Detecting Highfive dependencies (HIGHFIVE_USE_INSTALL_DEPS=NO)
CMake Error: install(EXPORT "xtensor-io-targets" ...) includes target "xtensor-io" which requires target "libdeps" that is not in any export set.
CMake Error in CMakeLists.txt:
  export called with target "xtensor-io" which requires target "libdeps" that
  is not in any export set.

* [tcl] [taskflow] Mark passing in CI.

ras0219 committed:
[vcpkg] Improve Json error messages (#12981)

* [vcpkg] Fix error reporting on json parse failure

* [vcpkg] Track manifest path for use in diagnostics

* [vcpkg] Use by-value for consumer API. Improve trailing comma diagnostic.

* [vcpkg] Track errors directly inside Json::Reader

* [vcpkg] Fixup use of .u8string()

Co-authored-by: Robert Schumacher <roschuma@microsoft.com>

ras0219 committed:
[vcpkg] Fix compilation on VS2015 (#13398)

Co-authored-by: Robert Schumacher <roschuma@microsoft.com>

PhoebeHui committed:
[boost] Remove invalid identifier !v120 (#13387)

jgehw committed:
[execute_process] Don't strip embedded semicolons (#12926)

NancyLi1013 committed:
[benchmark] Update to 1.5.1 and fix pkgconfig (#13427)

JackBoosY committed:
[libigl] Install extra headers when feature selected (#13424)

bialasjaroslaw committed:
[zeromq]Fix missing system version for mingw builds (#13408)

* Fix missing system version

System version is required to be 6.0 or higher for ZeroMQ to work correctly.

* Update ports/zeromq/CONTROL

Co-authored-by: Jack·Boos·Yu <47264268+JackBoosY@users.noreply.github.com>

DomClark committed:
[sord] Add new port (#13411)

* Add sord port

* Fix include directories

Neumann-A committed:
[mp-units] add new port (#13416)

* [mp-units] add new port

* fix config

bialasjaroslaw committed:
[boost-context] Fixes error with linker for 32-bit GNU assembler files targeting Windows (#13422)

* Patch from official boostorg/context. There is no official version with it yet.

* Apply patch globaly

It is better to check it now, for all triplets as [this script](https://github.com/microsoft/vcpkg/blob/master/scripts/boost/generate-ports.ps1) will regenerate portfile.cmake with patch for all of them not just for subset.

bialasjaroslaw committed:
[geos] Fix linking of geos for mingw (#13393)

* Fix linking of geos for mingw

Linking stage is failing due to the multiple definitions of the same symbol.
One way of fixing this is to disable inlining for small functions by passing DISABLE_GEOS_INLINE=ON for mingw builds.

* Bump the port version.

* [geos] Bump version

Co-authored-by: JackBoosY <yuzaiyang@beyondsoft.com>

mcmtroffaes committed:
[mp3lame] fix crt linkage (#13390)

* [mp3lame] fix crt linkage

* Update portfile.cmake

Co-authored-by: NancyLi1013 <46708020+NancyLi1013@users.noreply.github.com>

lebdron committed:
[vcpkg] improve s390x support (#13386)

Signed-off-by: Andrei Lebedev <lebdron@gmail.com>

JackBoosY committed:
[tmx] Fix build failure (#13401)

n4z4m3 committed:
[lerc] Add lerc port (#13358)

Co-authored-by: Nathan Mercer <nmercer@intermap.com>

traversaro committed:
[octomap] Update port to 1.9.5 (#13356)

ThomsonTan committed:
[msgpack] Update ports version from 3.2.0 to 3.3.0 (#13441)

ras0219 committed:
[assimp][irrxml][kubazip][poly2tri][polyclipping] Extract vendored dependencies (#13264)

* [assimp] Fix -config.cmake files

* [assimp] Fix linux build and provide backwards-compatible alias

* [assimp][irrxml][kubazip][poly2tri][polyclipping] Extract vendored dependencies

* [assimp] Use find_dependency(); add ZLIB dependency

* [polyclipping] Do not suffix upstream binaries

Co-authored-by: Robert Schumacher <roschuma@microsoft.com>
Co-authored-by: Stefano Sinigardi <stesinigardi@hotmail.com>

FranciscoPombal committed:
[libtorrent] Fix build on arm-linux community triplet (#13375)

* [libtorrent] Fix build on arm-linux community triplet

* [libtorrent] Add Port-Version to CONTROL file

Co-authored-by: NancyLi1013 <46708020+NancyLi1013@users.noreply.github.com>

* Add explicit dependeny on boost-pool for all triplets

Co-authored-by: Robert Schumacher <roschuma@microsoft.com>

Co-authored-by: NancyLi1013 <46708020+NancyLi1013@users.noreply.github.com>
Co-authored-by: Robert Schumacher <roschuma@microsoft.com>

Neumann-A committed:
[vcpkg|toolchain] add VCPKG_OVERLAY_(PORTS|TRIPLETS) to the toolchain (#13240)

* add VCPKG_OVERLAY_(PORTS|TRIPLETS) to the toolchain

* move everything into additional manifest params

* fix me not thinking

mfornace committed:
[taskflow] Update cpp-taskflow 2.2.0 to taskflow 2.6.0 (#13140)

* Update to 2.5.0

* Disable unit test building; 2.5.0 uses BUILD_TESTING flag

* Attempt to add /bigobj since I'm not sure how to disable building examples target

* Fix build error when building x64 targets

* Add more to the patch, remove sections with add_executable from CMakeLists.txt

* Guard with BUILD_TEST, put compile option in

* Disable CUDA

* Change name of package to taskflow

* Add patch so it's back to Cpp-Taskflow::Cpp-Taskflow...

* Update CONTROL

* Move - down to Port-Version.

* Add taskflow 2.6.0

* Update cpp-taskflow to 2.4.0 and disable test building

* Change 'cpp-taskflow' to merely depend on 'taskflow'.

* Remove osx skip.

* Fix up portfile and remove patch file since not needed for header-only

* Case fix for Linux

Co-authored-by: JackBoosY <yuzaiyang@beyondsoft.com>
Co-authored-by: Billy Robert O'Neal III <bion@microsoft.com>

mcmtroffaes committed:
[soxr] remove doc files from install (#13480)

JackBoosY committed:
[libigl] Re-fix install extra headers (#13471)

ras0219 committed:
[qhull] Update to 8.0.2 (#13445)

* [qhull] Extracted from Microsoft/vcpkg#12199

* [qhull] Update to 8.0.2 release

Co-authored-by: Stefano Sinigardi <stesinigardi@hotmail.com>
Co-authored-by: Robert Schumacher <roschuma@microsoft.com>

seanyen committed:
[urdfdom] Fixing up _IMPORT_PREFIX (#12795)

* Fixing up _IMPORT_PREFIX.

* update version.

* address the feedback.

gnaggnoyil committed:
[hffix] New port hffix library 1.0.0 (#13433)

ras0219 committed:
[pybind11] Fixup cmake information (#13443)

Co-authored-by: Stefano Sinigardi <stesinigardi@hotmail.com>

NancyLi1013 committed:
[shogun] Fix build error (#13425)

ras0219 committed:
[gtest] Remove autodetectable usage information (#13442)

Co-authored-by: Stefano Sinigardi <stesinigardi@hotmail.com>

JonLiu1993 committed:
[qwt] update to 6.1.5 (#12459)

* [qwt] update to 6.1.5

* delete patch file

Co-authored-by: Billy Robert O'Neal III <bion@microsoft.com>

LilyWangL committed:
[freeimage] Add freeimage-config.cmake (#12627)

* [freeimage] Add vcpkg-cmake-wrapper.cmake

* [freeimage] Prefer creating config.cmake instead of vcpkg-cmake-wrapper

Co-authored-by: Robert Schumacher <roschuma@microsoft.com>

martin-s committed:
[vcpkg] Fix issue #12745. (#12746)

id6tm-mteterel committed:
[d3dx12] Add new port (#13360)

* [d3dx12] new port

* Apply suggestions from code review

Co-authored-by: NancyLi1013 <46708020+NancyLi1013@users.noreply.github.com>

* Update portfile.cmake

Use the correct d3dx12.h that does not require the very latest Windows SDK

* Apply suggestions from code review

Co-authored-by: Robert Schumacher <roschuma@microsoft.com>

* Update ports/d3dx12/portfile.cmake

* [d3dx12] Download license file

Co-authored-by: NancyLi1013 <46708020+NancyLi1013@users.noreply.github.com>
Co-authored-by: Robert Schumacher <roschuma@microsoft.com>

BillyONeal committed:
[vcpkg] Merge the vcpkg metadata uploader into the vcpkg binary (#13421)

BillyONeal committed:
Fix boostrap on VS2019. (#13489)

menduz committed:
[yoga] Update to 1.18.0 (#13515)

* Update CONTROL

* Update portfile.cmake

* Update add-project-declaration.patch

strega-nil committed:
[vcpkg manifest] Add documentation! (#13488)

* [vcpkg docs] add docs for manifest files

These are just for the maintainer docs,
not user docs.

* [vcpkg] EBNF-ify platform expression parsing

this modifies nothing about what strings are accepted or rejected,
it just moves stuff around.

also adds tests.

* [vcpkg docs] add manifest mode example

* [wip] docs for augustin

also fix tabs

* [vcpkg manifest] switch to using maps for features

* Apply suggestions from code review

* un-experimentize format-manifest

* flesh out the user manifest mode docs

* CRs

* billy CRs

* final personal pass-thru

c72578 committed:
[tool-meson] Update to 0.55.3 (#13497)

ras0219 committed:
[vcpkg] Add metrics for binarycaching feature (#13486)

Co-authored-by: Robert Schumacher <roschuma@microsoft.com>

kevinlul committed:
[sqlite3] Update to 3.33.0 (#13501)

JackBoosY committed:
[sdl2] Fix pkgconfig: missing system lib dbus-1 (#13520)

xieyubo committed:
[ffmpeg:arm64-linux] Fix bug #13511: [ffmpeg:arm64-linux] build failure. (#13512)

Root cause:
In `ffmpeg/portfile.cmake`, `--target-os=win32` is applied to all arm or arm64 targets.
This is wrong.

Fix:
Only apply "--target-os=win32" if VCPKG_TARGET_IS_WINDOWS.

Verify:
Run `./vcpkg install ffmpeg` on arm64-linux host, make sure the generated libraries are *.a not *.lib.

rkesters committed:
[croncpp] Added croncpp port (#13359)

* Added croncpp port

* Update croncpp for windows failure

* Update ports/croncpp/portfile.cmake

Co-authored-by: Robert Schumacher <roschuma@microsoft.com>

* Update ports/croncpp/portfile.cmake

Co-authored-by: Robert Schumacher <roschuma@microsoft.com>

* Update ports/croncpp/CONTROL

Co-authored-by: Robert Schumacher <roschuma@microsoft.com>

* Update ports/croncpp/portfile.cmake

Co-authored-by: Robert Schumacher <roschuma@microsoft.com>

* Update ports/croncpp/portfile.cmake

Co-authored-by: Robert Schumacher <roschuma@microsoft.com>

* Update ports/croncpp/portfile.cmake

Co-authored-by: Robert Kesterson <robert.d.kesterson@leidos.com>
Co-authored-by: Robert Schumacher <roschuma@microsoft.com>

stijnherfst committed:
[qt-advanced-docking-system] update to 3.6.1 (#13340)

* update to 3.6.1

* remove unneeded changes

* fix qt5-x11extras linux dependency

* add dependency for osx

* make requested changes

* change hash

n4z4m3 committed:
[triangle] Add new port (#13322)

* New Port : libigl-triangle

* Update ci.baseline.txt for libigl-triangle port

* Update ports/libigl-triangle/CONTROL

Unnecessary Port-Version

Co-authored-by: NancyLi1013 <46708020+NancyLi1013@users.noreply.github.com>

* Update ports/libigl-triangle/portfile.cmake

Unnecessary inclusion of vcpkg_common_functions

Co-authored-by: NancyLi1013 <46708020+NancyLi1013@users.noreply.github.com>

* Update ports/libigl-triangle/portfile.cmake

Use README from source as the copyright file

Co-authored-by: NancyLi1013 <46708020+NancyLi1013@users.noreply.github.com>

* Updates to libigl-translate port

* Update libigl-triangle port

* Update ports/libigl-triangle/CONTROL

Co-authored-by: NancyLi1013 <46708020+NancyLi1013@users.noreply.github.com>

* Update ports/libigl-triangle/portfile.cmake

Co-authored-by: NancyLi1013 <46708020+NancyLi1013@users.noreply.github.com>

* Make the header copy properly and add CMake targets

* Utilize original sources for triangle library and rename to triangle

* Build triangle executable and add it to tools

* Fix Linux build which requires linking to m library for math functions

* Update scripts/ci.baseline.txt

Co-authored-by: NancyLi1013 <46708020+NancyLi1013@users.noreply.github.com>

* Instead of defining INT_PTR, use uintptr_t from stdint.h
Also, define FLOAT and VOID in the header so the user of the library does not have to define them

* Make sure the port works and can be used

* Remove the use of the SINGLE define for switching the REAL define to be float or double. Also make the define for VOID be void rather than replacing all VOID with void in order to reduce the patch size.

Co-authored-by: Nathan Mercer <nmercer@intermap.com>
Co-authored-by: NancyLi1013 <46708020+NancyLi1013@users.noreply.github.com>
Co-authored-by: NancyLi1013 <lirui09@beyondsoft.com>

traversaro committed:
[dartsim] Add new port (#13320)

* [dartsim] Add new port

* Update vcpkg.json

* Update vcpkg.json

* Update vcpkg.json

ras0219 committed:
[vcpkg-docs] Add docs/users/binarycaching.md (#13517)

* [vcpkg-docs] Add docs/users/binarycaching.md

* [vcpkg] Address PR Comments

Co-authored-by: Robert Schumacher <roschuma@microsoft.com>

traversaro committed:
[dartsim] Fix version in vcpkg.json (#13550)

coryan committed:
[google-cloud-cpp] Update to v1.18.0 release (#13552)

* [google-cloud-cpp] Update to v1.18.0 release

* Update portfile.cmake

Add unsupported message

Co-authored-by: NancyLi1013 <46708020+NancyLi1013@users.noreply.github.com>

eao197 committed:
[json_dto] Update to v.0.2.10 (#13535)

* json_dto updated to v.0.2.10

* Update portfile.cmake

Co-authored-by: NancyLi1013 <46708020+NancyLi1013@users.noreply.github.com>

riverar committed:
[vcpkg_docs] Fix feature flags option typo (#13578)

hanseuljun committed:
[opus] Fix for arm64-ios (#13568)

* Add SYSTEM_LIBRARIES m to vcpkg_fixup_pkgconfig() in opus/portfile.cmake.

* [opus] Update CONTROL

Co-authored-by: wangli28 <wangli28@beyondsoft.com>

danielaparker committed:
[jsoncons] Update to version 0.156.1 (#13595)

strega-nil committed:
[vcpkg manifest-mode] Fix parallel msbuild failures (#13589)

This changes the vcpkg msbuild targets file to wait for the lock, rather than failing immediately

here-mikelley committed:
[vcpkg] Keep sym links in Linux zip cache files (#13609)

* [vcpkg] Add '-y' to binary caching zip command on non-Windows platforms

PhoebeHui committed:
[inja/kd-soap/fastrtps/graphite2/jsonnet] Update to latest release revision (#13584)

* [inja/kd-soap/fastrtps/graphite2/jsonnet] Update to latest release revision

* Fix license path for fastrtps and remove bin with static build for kd-soap

glynos committed:
[skyr-url] Changed version number to 1.13.0 (#13617)

evpobr committed:
[libsndfile] Update to 1.0.30 (#13616)

* Update to latest release
* Remove patch merged to upstream
* Add vcpkg_fixup_pkgconfig() call
* Do not use deprecated option `ENABLE_STATIC_RUNTIME`

Ryan-rsm-McKenzie committed:
[xbyak] Update port to 5.97 (#13553)

AenBleidd committed:
[sentry-native] Update port to 0.4.2 (#13546)

* [sentry-native] Update port to 0.4.2

Remove custom fix for msvc 19.27 because of the fix that was introduced by port authors here: https://github.com/getsentry/sentry-native/commit/f1437333d4ea1eca2f8a3664d57efff081b4dad6

Signed-off-by: Vitalii Koshura <lestat.de.lionkur@gmail.com>

* Fix review comments

Signed-off-by: Vitalii Koshura <lestat.de.lionkur@gmail.com>

myd7349 committed:
[unqlite] Add new port (#13544)

BillyONeal committed:
[vcpkg] Add build scripts to produce signed vcpkg binaries. (#13508)

NancyLi1013 committed:
[Many ports] Update to the latest version or commit (#13560)

* [Many ports] Update to the latest version or commit

* [libplist] Update version
[libsync] Update patch format

* Update as review suggestions

* [libplist] Revert changes

FabienPean committed:
[polyclipping] Fix cmake patch for include directory (#13668)

alek-d committed:
[scintilla] Add static linking to Scintilla (#13529)

* Add static linking to Scintilla

* Add Port-Version to CONTROL

* Split library linkage from crt linkage

* Update CI baseline

* Remove vcpkg_copy_pdbs()

samyy321 committed:
[nanodbc] Update to 2.13.0 (#13662)

mcmtroffaes committed:
[ffnvcodec] only supports windows and linux (#13638)

JonLiu1993 committed:
[mimalloc] Update to 1.6.4 (#13629)

* [mimalloc] Update to 1.6.4

* deletecibaseline

BillyONeal committed:
[vcpkg] Fix the case of current_path() before use on Windows. (#13537)

* Fix the case of current_path() before use on Windows.

This is a better solution than that I tried in https://github.com/microsoft/vcpkg/pull/13144 -- rather than trying to cannoicalize the path completely, which would destroy symlinks etc; this form  calls FindFirstFile on each path element to get the real case from the filesystem.

Fixes #13105.

(Note the wrong case in the CWD in the prompt, but the correct case in all the output:)

PS C:\DeV\vcPKG> .\vcpkg.exe install curl
Computing installation plan...
The following packages will be built and installed:
    curl[core,non-http,schannel,ssl,sspi,winssl]:x86-windows
  * zlib[core]:x86-windows
Additional packages (*) will be modified to complete this operation.
Detecting compiler hash for triplet x86-windows...
Starting package 1/2: zlib:x86-windows
Building package zlib[core]:x86-windows...
Using cached binary package: C:\Users\billy\AppData\Local\vcpkg/archives\c6\c61dd1bcc23348934c55f4ced77f1e4b0f353394.zipBuilding package zlib[core]:x86-windows... done
Installing package zlib[core]:x86-windows...
Installing package zlib[core]:x86-windows... done
Elapsed time for package zlib:x86-windows: 62.26 ms
Starting package 2/2: curl:x86-windows
Building package curl[core,non-http,schannel,ssl,sspi,winssl]:x86-windows...
Could not locate cached archive: C:\Users\billy\AppData\Local\vcpkg/archives\9f\9fa16d473c9539e9ea7ab3922eff46618f3a4c4b.zip
-- Downloading https://github.com/curl/curl/archive/9d954e49bce3706a9a2efb119ecd05767f0f2a9e.tar.gz...
-- Extracting source C:/Dev/vcpkg/downloads/curl-curl-9d954e49bce3706a9a2efb119ecd05767f0f2a9e.tar.gz
-- Applying patch 0002_fix_uwp.patch
-- Applying patch 0004_nghttp2_staticlib.patch
-- Applying patch 0005_remove_imp_suffix.patch
-- Applying patch 0006_fix_tool_depends.patch
-- Applying patch 0007_disable_tool_export_curl_target.patch
-- Applying patch 0009_fix_openssl_config.patch
-- Applying patch 0010_fix_othertests_cmake.patch
-- Applying patch 0011_fix_static_build.patch
-- Using source at C:/Dev/vcpkg/buildtrees/curl/src/767f0f2a9e-91d24adee1.clean
-- Configuring x86-windows
-- Building x86-windows-dbg
-- Building x86-windows-rel
-- Installing: C:/Dev/vcpkg/packages/curl_x86-windows/share/curl/curl-config
-- Installing: C:/Dev/vcpkg/packages/curl_x86-windows/share/curl/vcpkg-cmake-wrapper.cmake
-- Installing: C:/Dev/vcpkg/packages/curl_x86-windows/share/curl/copyright
-- Performing post-build validation
-- Performing post-build validation done
Stored binary cache: C:\Users\billy\AppData\Local\vcpkg/archives\9f\9fa16d473c9539e9ea7ab3922eff46618f3a4c4b.zip
Building package curl[core,non-http,schannel,ssl,sspi,winssl]:x86-windows... done
Installing package curl[core,non-http,schannel,ssl,sspi,winssl]:x86-windows...
Installing package curl[core,non-http,schannel,ssl,sspi,winssl]:x86-windows... done
Elapsed time for package curl:x86-windows: 1.299 min

Total elapsed time: 1.33 min

The package curl:x86-windows provides CMake targets:

    find_package(CURL CONFIG REQUIRED)
    target_link_libraries(main PRIVATE CURL::libcurl)

PS C:\DeV\vcPKG>

* Fix *nix builds.

* PR feedback.

* Update toolsrc/src/vcpkg/base/files.cpp

Co-authored-by: ras0219 <533828+ras0219@users.noreply.github.com>

iainchesworth committed:
[tinyutf8] Updating to v4.0.2 (#13706)

Update DuffsDevice/tinyutf8 library from v3.0.1 to v4.0.2

JackBoosY committed:
[documentation] Fix manifest link (#13685)

* [document] Fix manifest docs

* Sync to Chinese document

* add empty line

* Fix manifest link

NancyLi1013 committed:
[orc] Update to the latest version (#13705)

* [orc] Update to the latest version

* Update patch format

pratikpc committed:
[vcpkg] Show Empty Object on vcpkg --list when used with --x-json (#13667)

Thanks to @strega-nil for recommending using Json::stringify over my initial idea of simply printing an empty object

Closes #13637

janholt committed:
[apr-util] use find_package(expat) (#13663)

Co-authored-by: janholt <janholt@gmail.com>

mcmtroffaes committed:
[libvpx] fix cmake target for linux (#13636)

ctabin committed:
[libzip] Update to libzip 1.7.3 (#13625)

* Update to libzip 1.7.3

* [libzip] Update CONTROL

Co-authored-by: Cédric Tabin <cedric.tabin@gmail.com>
Co-authored-by: wangli28 <wangli28@beyondsoft.com>

alexander-smyslov committed:
[qt5] Install Qt5 Network SSL dependencies (#13599)

* Add files via upload

* [new port] avro-cpp

* add boost-format

* remove comments

* remove comments

* fix copy ssl libraries

* fix

* bump version

Co-authored-by: Alexander Smyslov <aleksandr.smyslov@libertexgroup.com>
Co-authored-by: Smyslov Alexander Nikolayevich <aleksandr.smyslov@fxclub.com>
Co-authored-by: JackBoosY <yuzaiyang@beyondsoft.com>

hanseuljun committed:
[libvpx] allow arm64 as target architecture (#13569)

* [libvpx] allow arm64 as target architecture

* [libvpx] bump port version to 2

Farwaykorse committed:
[Ninja] update to 1.10.1 (#13562)

* [Ninja] update to 1.10.1

* [Ninja] update to 1.10.1 (find_acquire_program)

Co-authored-by: Billy Robert O'Neal III <bion@microsoft.com>

alek-d committed:
[scintilla] Update to 4.4.5 (#13665)

* Add static linking to Scintilla

* Add Port-Version to CONTROL

* Split library linkage from crt linkage

* Update CI baseline

* [scintilla] Update to 4.4.5

* Remove vcpkg_copy_pdbs()

* [scintilla] Add supports

Co-authored-by: wangli28 <wangli28@beyondsoft.com>

myd7349 committed:
[nng] Update to 1.3.2 (#13545)

* [nng] Update to 1.3.2

[nng] Add tools feature

* [nng] Let's give ninja a second chance

* Update ports/nng/portfile.cmake

Co-authored-by: Robert Schumacher <roschuma@microsoft.com>

Co-authored-by: NancyLi1013 <46708020+NancyLi1013@users.noreply.github.com>
Co-authored-by: Robert Schumacher <roschuma@microsoft.com>

lebdron committed:
[vcpkg, libpq] separate static and shared library installations (#13491)

* [vcpkg] add missing Makefile flag for install step

Signed-off-by: Andrei Lebedev <lebdron@gmail.com>

* [libpq] add custom Makefile to install only required targets

Signed-off-by: Andrei Lebedev <lebdron@gmail.com>

mcmtroffaes committed:
[ffmpeg] fix feature dependencies (#13502)

* [ffmpeg] fix feature dependencies

* do not pull in default depedencies when specifying nonfree/gpl
* explicitly mark postproc as gpl
* add required dependencies for ffmpeg, ffplay, and ffprobe
* remove defunct ffserver feature (this feature was removed in ffmpeg a while ago)

* [ffmpeg] fix feature dependencies for avformat and avdevice

* [ffmpeg] Bump port-version to 25

Co-authored-by: Robert Schumacher <roschuma@microsoft.com>

mcmtroffaes committed:
[ffmpeg] fix linking with iconv feature (#13481)

* [ffmpeg] fix linking with iconv feature

* [ffmpeg] use official iconv cmake target (broken for now, needs fix in libiconv, see #11776)

* [ffmpeg] fix typo in theora configuration

* [ffmpeg] switch to $<NOT:$<CONFIG:Debug>> for selecting release libraries

* [ffmpeg] use find_dependency(Iconv) on macos to better handle downstream REQUIRED

* Bump port version.

* [ffmpeg] remove iconv dependency on mac when it's not needed

Patch originally provided by @cenit

Neumann-A committed:
[x264] Fix x64-windows-static (#13450)

* try fixing x264

* fix remaining issues

* Remove unnecessary COPY_SOURCE

DomClark committed:
[lv2] Add port (#13462)

cnbatch committed:
[libtins] Update to version 4.3 (#13681)

* [libtins] Upgrade version to 4.3

And enables C++11 support

* [libtins] Upgrade version to 4.3

And enables C++11 support

strega-nil committed:
[vcpkg macos ci] Switch to using our own base boxes, to fix bringing up mac machines (#13619)

* [vcpkg ci:osx] Remove brew install

* add instructions for creating a new vagrant box

* fix the vagrant scripts for the new box

* finish fixing the setup

* [mecab jxrlib] fix ports for CI

mecab needed to use an actual ref that wasn't master,
and jxrlib needed a patch for xcode 12 CLTs.

Additionally, this fixes the mecab version to be a date, the date of the last commit,
since `1.0` is not the correct version (mecab doesn't have released versions)

* [many ports] fix compile with Xcode 12 CLTs

This mostly means fixing errors on implicit-function-declaration,
and removing some Werrors
* alac-decoder
* apr
* argtable2
* arrow
* hyperscan
* mcpp
* minizip
* mosquitto
* stormlib

* [many ports] even more Xcode 12 CLT fixes

* [jxrlib darknet] fix the last ports! (hopefully)

* CRs, plus minor wip changes to osx scripts

UnixY2K committed:
fix broken link

UnixY2K committed:
Improved translation

UnixY2K committed:
Improved translation
